react/.storybook/main.js

```diff
@@ -8,29 +9,55 @@ module.exports = {
     '@storybook/addon-viewport',
     '@storybook/addon-storysource',
     '@storybook/addon-a11y',
+    {
```

Comment:
```suggestion
    '@storybook/preset-create-react-app'
```

Reply:
can't do multi-line suggestion, but since there's no additional config, can you change to match the others